dynamodb-quickstart/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+quarkus.dynamodb.region=eu-central-1
```

Comment:
I'm a dynamodb noob, but I'm wondering why you need to specify a region if the endpoint is targeting localhost?

Reply:
If region is not specified, then default region provider chain fires that looks up for region in env properites, system properties or aws profile files on local file system. Basically even local instance require region to be specified as it's part of the HTTP request to the dynamodb (local instance requires it too, but it can be any region)